homeassistant/components/balboa/translations/fr.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connexion au module Wi-Fi Balboa",
+        "data": {
+          "host": "H\u00F4te",
+          "name": "Nom"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "\u00c9chec de la connexion, veuillez r\u00e9essayer",
+      "unknown": "Erreur inattendue"
+    },
+    "abort": {
+      "already_configured": "L'appareil est d\u00e9j\u00E0 configur\u00e9"
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "sync_time": "Gardez l'heure du module Wi-Fi Balboa synchronis\u00e9e avec Home Assistant"
+        }
+      }
+    }
+  }
+}
```

Comment:
Only en.json should be uploaded (other languages are added automatically by the translator)
```suggestion
```